client/src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
Optional, consider cleaning up files such as 
- Removing ./index.css
- Removing reportWebVitals (unless you have plans to use it)
- etc.